### ISSUE TITLE
About the loss functions used in system/neus.py.

### ISSUE THREAD

**Issue by ZhouWeikun** [bennyguo/instant-nsr-pl#74]
First, thanks for your work.
But i have some questions about the functions you used in neus:
1.`loss_sparsity = torch.exp(-self.config.system.loss.sparsity_scale * out['sdf_samples'].abs()).mean()`
2.`loss_opaque = binary_cross_entropy(opacity, opacity)`
Could you give more info about them, or  papers related to them?

**Comment by bennyguo** [comment]
Hi! The sparsity loss is proposed by SparseNeuS and is used to remove unnecessary occupancy in the few-shot setting:

![image](https://github.com/bennyguo/instant-nsr-pl/assets/19284678/afadd57c-879e-4003-9b4e-5ed01863d642)

I tried training with this loss but found no obvious difference in the dense-capture setting (e.g., NeRF-Synthetic).

The opaque loss is to encourage the opacity to be either 0 or 1. It is used in my [VMesh paper](https://arxiv.org/abs/2303.16184):

<img width="486" alt="image" src="https://github.com/bennyguo/instant-nsr-pl/assets/19284678/497e51f0-9965-4ec5-a93f-7b49473ae828">

I found this loss to be improving surface quality for NeuS in some scenarios.

Hope this helps!
